.gitignore

```diff
@@ -5,10 +5,16 @@ doc/tag/*
 # Ignore CMake build directory.
 build/
 
-#Ignore build artifacts
+# Ignore build artifacts
 *.o
 
 # Ignore code coverage artifacts
 *.gcda
 *.gcno
 *.gcov
+
+# Ignore CBMC proof artifacts
+**/cbmc/proofs/**/gotos
+**/cbmc/proofs/**/html
+**/cbmc/proofs/**/logs
+**/cbmc/proofs/**/TAGS-*
```

Comment:
Could add `**/cbmc/proofs/**/report` which is the equivalent of html for the new viewer.

Reply:
Added in mqtt/proofs branch. Will be in the next PR.